Makefile

```diff
@@ -105,6 +105,10 @@ else
 
 pytest: all
 	@echo "=== Running python tests ==="
+ifeq ($(shell which pytest-3),)
```

Comment:
Thanks, but we don't want to depend on having which installed.

Instead, you can check this by looking at PYTESTCMD: if it's /bin/true, then we didn't find pytest-3 and can report the message